ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -320,7 +330,13 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
       {mainGeneId && <VariantGenes mainGeneId={mainGeneId} showMainGene={showMainGene} variant={variant} />}
       {(mainGeneId && Object.keys(variant.transcripts || {}).length > 1) && <VerticalSpacer height={10} />}
       <LargeText>
-        <b><UcscBrowserLink variant={variant} includeEnd={!!variant.svType} /></b>
+        <b><UcscBrowserLink variant={variant} includeEnd={!!svType && !endChrom} /></b>
```

Comment:
this is the one place where I think it is appropriate to check if `svType` is a translocation, as you are correct that we want to show translocations differently even if they are within the same chromosome